requirements.txt

```diff
@@ -1,3 +1,4 @@
+jsonconversion==0.2.12
```

Comment:
This isn't part of the built-in JSON library?

Reply:
This library uses the built-in JSON library and wraps the class logic easily so that we can overwrite it with the logic we need.